src/utils/todoFilter.ts

```diff
@@ -0,0 +1,36 @@
+import { ETodoStatus, Todo } from '../types/Todo';
+
+interface ITodosQuery {
+  inputField?: string;
+  filteredBy?: ETodoStatus;
+}
+
+export function getPreparedTodos(todos: Todo[], {
+  inputField = '',
+  filteredBy = ETodoStatus.All,
+}: ITodosQuery) {
+  let preparedTodos = [...todos];
```

Comment:
The `filter` method returns a copy of the array to which it is called, so you don't have to do it manually